lib/controllers/createController.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var path = require('path');
+
+module.exports = function (apiPlatformService, fileSystemRepository,
+    workspaceRepository) {
+  return {
+    create: create
+  };
+
+  function create(newApi) {
+    return apiPlatformService.createAPI(newApi.bizGroup, newApi.api,
+        newApi.apiVersion, newApi.rootRamlPath)
+      .then(function (api) {
+        return workspaceRepository.get()
+          .then(function (workspace) {
+            return fileSystemRepository.getFileHash(newApi.rootRamlPath)
+              .then(function (rootRamlHash) {
+                workspace.files = [
+                  {
+                    path: path.sep + newApi.rootRamlPath,
```

Comment:
I'm not sure about using path.sep as this path needs to be compared with API Platforms paths and I think it always use '/'